src/backend/marsha/core/migrations/0016_auto_20191001_0812.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 2.2.5 on 2019-10-01 08:12
+
+import json
+import re
+
+from django.conf import settings
+from django.db import migrations
+
+import boto3
+
+
+def escape_timed_text_tracks(apps, schema_editor):
+    if not settings.SHOULD_RUN_TIMEDTEXT_TRACKS_ESCAPING_MIGRATION:
+        return
+
+    marsha_s3 = boto3.client(
+        "s3",
+        region_name=settings.AWS_S3_REGION_NAME,
+        aws_access_key_id=settings.AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=settings.AWS_SECRET_ACCESS_KEY,
+    )
+    marsha_lambda = boto3.client(
+        "lambda",
+        region_name=settings.AWS_S3_REGION_NAME,
+        aws_access_key_id=settings.AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=settings.AWS_SECRET_ACCESS_KEY,
+    )
+    base_aws_name = settings.AWS_SOURCE_BUCKET_NAME.rstrip("-source")
+    lambda_function_name = f"{base_aws_name}-encode"
```

Comment:
these 2 lines :cry:

Reply:
What should I do ?

Reply:
Maybe introduce a setting `AWS_BASE_NAME` and build the other ones from it? This has the advantage of being more DRY as well...
```python
AWS_BASE_NAME = "staging-marsha"
AWS_SOURCE_BUCKET_NAME = f"{AWS_BASE_NAME:s}-source"
AWS_STATIC_BUCKET_NAME = f"{AWS_BASE_NAME:s}-static"
AWS_LAMBDA_ENCODE_NAME = f"{AWS_BASE_NAME:s}-encode"
```

Reply:
I already think about this king of solution but it introduces a BC break I think :-)

Reply:
I don't see why as long as the existing settings are still there, still set at the same value and can still be overidden by an env variable.

We must update the env doc but I don't think it is a BC.

Reply:
hmm ok we can decide it's a new environment variable, keep the old one. This release will be a minor in this case (`3.1.0`)

Reply:
I introduce a new setting `AWS_BASE_NAME`. You can review all the PR I think.